go/libkb/pgp_key.go

```diff
@@ -607,6 +609,13 @@ func (k *PGPKeyBundle) CheckFingerprint(fp *PGPFingerprint) error {
 }
 
 func (k *PGPKeyBundle) SignToString(msg []byte) (sig string, id keybase1.SigID, err error) {
+	if k.Entity.PrivateKey == nil || k.Entity.PrivateKey.PrivateKey == nil {
```

Comment:
At the very least, I think a check for the existence of gpg should be done before heading down this path.

I'm interested in your answers to @maxtaco's questions above, too, as this is a fairly big change.  We would have to update our help docs as well that explain when we use gpg, when we use internal libraries to explain to the user that we would be using gpg in this case.